src/Microsoft.AspNet.Authentication/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,7 @@ public static class AuthenticationServiceCollectionExtensions
     {
         public static IServiceCollection AddAuthentication([NotNull] this IServiceCollection services)
         {
+            services.AddCaching();
```

Comment:
Yeah this is tricky then, if it requires services, as it's going to require its own AddOIDC extension then?